packages/server-core/src/projects/project/project-helper.ts

```diff
@@ -1426,8 +1426,8 @@ export const createExecutorJob = async (
   })
 }
 
-export const copyDefaultProject = () => {
-  deleteFolderRecursive(path.join(projectsRootFolder, `default-project`))
+export const copyDefaultProject = async () => {
```

Comment:
usage of `copyDefaultProject` has not been changed to be awaited. it's called in `seeder,ts` and `project-helper.ts`

Reply:
this is likely what is causing checks to fail

Reply:
Good catch, I've already changed any call to copyDefaultProject to await for the promise to resolve, rebased, squached and pushed the updates, though while running the tests after addressing your comments I'm still getting the same bunch of errors I got yesterday, my assumption after analyzing the error stacks has been that these are unrelated to my modifications since in the stack hasn't include any file or function I created or modified, but after addressing your comments, I thought maybe these errors actually are very indirectly affected by my code, so I just tried running the tests from the dev branch and verify if there is any difference just to learn that I get the exact same errors I get while running the tests in my branch, that confirmed my assumption that if within the error stacks my code isn't mentioned, I also get the same errors on the dev branch, then these errors aren't caused by it.... Thoughts?